### ISSUE TITLE
Unable to replicate SCNet pre-trained model SDR values using same config

### ISSUE THREAD

**Issue by KaiMikkelsen** [ZFTurbo/Music-Source-Separation-Training#143]
Hello,

I am trying to reproduce the results obtained using SCNet and am using the same config provided in the pretrained_models.md file (https://github.com/ZFTurbo/Music-Source-Separation-Training/blob/main/docs/pretrained_models.md) .

The best SDR I can get when training on the MUSDB18HQ dataset is an average of ~7, and the pre_trained model and information listed with the config mentions SDR values closer to 9 are possible with that config and the MUSDB18HQ dataset.

<img width="996" alt="Image" src="https://github.com/user-attachments/assets/f4c5f705-81e1-4ce3-9aa9-bf7e6c7513b4" />

Has anyone tried to retrain SCNet with that config and been able to reproduce higher SDR values?

Thanks!

**Comment by ZFTurbo** [comment]
This config wasn't used for training - it's suitable only for inference.
Model were trained using this repository by author:
https://github.com/starrytong/SCNet?tab=readme-ov-file#training

**Comment by KaiMikkelsen** [comment]
> This config wasn't used for training - it's suitable only for inference. Model were trained using this repository by author: https://github.com/starrytong/SCNet?tab=readme-ov-file#training

Is that the case with all the pre-trained models? The numbers and checkpoints provided are all from the original repositories?

**Comment by ZFTurbo** [comment]
Most of models wasn't trained from scratch here. I remember that I trained BSRoformer on MUSDB18 using 8*A100:
https://github.com/ZFTurbo/Music-Source-Separation-Training/releases/download/v1.0.12/config_bs_roformer_384_8_2_485100.yaml
https://github.com/ZFTurbo/Music-Source-Separation-Training/releases/download/v1.0.12/model_bs_roformer_ep_17_sdr_9.6568.ckpt

But I restarted training several times, so you can't just use single config. As I remember order:
1) Dataset type: 1, Augm: on
2) Dataset type: 1, Augm: off
3) Dataset type: 4, Augm: off

**Comment by starrytong** [comment]
@ZFTurbo repository uses automatic mixed precision (AMP) training by default, but I trained the SCNet model using full-precision FP32, which naturally resulted in degraded performance. You can try training the SCNet-large model directly.
@KaiMikkelsen

**Comment by KaiMikkelsen** [comment]
Sounds good thank you @starrytong @ZFTurbo. I will try both of those approaches.

**Comment by hollandloprabbit** [comment]
> Most of models wasn't trained from scratch here. I remember that I trained BSRoformer on MUSDB18 using 8*A100: https://github.com/ZFTurbo/Music-Source-Separation-Training/releases/download/v1.0.12/config_bs_roformer_384_8_2_485100.yaml https://github.com/ZFTurbo/Music-Source-Separation-Training/releases/download/v1.0.12/model_bs_roformer_ep_17_sdr_9.6568.ckpt
> 
> But I restarted training several times, so you can't just use single config. As I remember order:
> 
> 1. Dataset type: 1, Augm: on
> 2. Dataset type: 1, Augm: off
> 3. Dataset type: 4, Augm: off

Hi, I would like to ask for some clarification. When it says "not training from scratch" here, does it mean that model needs to adjust the training configuration multiple times during training? During multiple rounds of training, were there any modifications besides data augmentation? For example, were parameters like learning rate or chunk size also changed? Thank you!
